### PR TITLE
skydive: remove it from nixpkgs

### PR DESCRIPTION
**Pull request by nlewo**
The current Skydive version can not be build with a recent Go version
and the maintainer (@nlewo) is no longer interested in maintaining it.

###### Motivation for this change

The pr #68195 is removing Go 1.11 which is required to compile the current Skydive version.
I've tried to package the latest version, but the build is tricky, it requires several missing dependencies and i'm no longer using Skydive. 

However, if someone is interested in maintaining it, I can help to update it (i have something almost working but no more time to work on it).

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -13562,12 +13562,6 @@ in
     sdnotify-wrapper = callPackage ../os-specific/linux/sdnotify-wrapper { };
   };
 
-  skydive = callPackage ../tools/networking/skydive {
```
Comment (worldofpeace) [inline pkgs/top-level/all-packages.nix]: Please add a `throw` to `aliases.nix`. Examples there.

**Comment by lheckemann** [comment]
Since we don't have a maintainer, we should probably backport this to 19.09 as well, right?

**Comment by nlewo** [comment]
@lheckemann yes, I could do it.

**Review by worldofpeace — APPROVED** [review]

**Comment by worldofpeace** [comment]
backported in fcd7d6ad4189216bef0267fb43bb95bed8e14e9c

**Pull request merged**